gulpfile.js

```diff
@@ -26,7 +26,20 @@ const md = new Remarkable({
 });
 
 const packageJSON = require('./package.json');
-const vendorModules = Object.keys(packageJSON.dependencies);
+const excludedVendorModules = [
+  'babel-polyfill',
+  'l20n'
+];
+const includeVendorModules = [
+  'babel-polyfill/browser',
+  'l20n/dist/compat/web/l20n'
+];
+const vendorModules = Object.keys(packageJSON.dependencies)
+  .filter(name => {
+    if (excludedVendorModules.indexOf(name) !== -1) { return false; }
+    return true;
```

Comment:
FWIW, there may also be an `.includes()` API, depending on our Node+Babel env:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

> _The `includes()` method determines whether an array includes a certain element, returning `true` or `false` as appropriate._